app/main/index.ts

```diff
@@ -108,10 +110,12 @@ function createMainWindow(): Electron.BrowserWindow {
 	win.setTitle('Zulip');
 
 	win.on('enter-full-screen', () => {
+		ViewManager.fixBounds();
```

Comment:
Can you document the bound thing here and what it do etc?

Reply:
I have documented this here. https://github.com/zulip/zulip-desktop/blob/e4f025166f882ffa9bd6fcde73fe19d4d7572d76/app/main/viewmanager.ts#L106